src/python/deepgnn/graph_engine/snark/client.py

```diff
@@ -382,6 +434,39 @@ def node_sparse_features(
 
         return py_cb.indices, py_cb.values, py_cb.dimensions
 
+    def node_string_features(
+        self, nodes: np.array, features: np.array, dtype: np.dtype
```

Comment:
Why do we not just make this the default? It seems easier for users and should be just as fast?